src/components/NavbarSection/CartPopup/CartPopupButton.tsx

```diff
@@ -12,38 +12,23 @@ const CartPopupButton: React.FC<CartPopupButtonProps> = (props) => {
     const cartCount = props;
```

Comment:
you can take an approach like
`const { value } = props;`
So you don't need the period to access the props.

Reply:
Cool, will implement that 😃 Thanks for the tip 👍🏻